include/aspect/postprocess/visualization.h

```diff
@@ -124,6 +124,13 @@ namespace aspect
       class Interface
       {
         public:
+          /**
+           * Constructor. The constructor takes as argument the physical
+           * units of the quantity (scalar or vector-valued) computed by
+           * derived classes.
+           */
+          Interface (const std::string &physical_units);
```

Comment:
If you make this optional we could merge this PR separately from later implementations for individual plugins. Or is this PR not intended to be merged, but we focus on #4244 instead?

```suggestion
          Interface (const std::string &physical_units = "");
```

Reply:
This was just the basis for #4244. I should have closed it right away after @ahollyday started his work.